.gitignore

```diff
@@ -8,3 +8,4 @@ package-lock.json
 .nyc_output
 coverage
 *.generated.css
+.next
```

Comment:
is your code editor not set to have an empty line at the end of the file @dmtrKovalenko ?

Reply:
No, I propose to add an `.editorconfig` and prettier to not care about such things